### DISCUSSION TITLE
[QUESTION] - What does oml-luxor extension provide?

### DISCUSSION THREAD

**Discussion by mprather** [opencaesar/platform#16] [Q&A]
## Description
The readme is somewhat geared toward the developer that may wish to contribute to the project. However, the only info in the readme that is provided to a non-dev is "An extension to support OML in VSCode-based IDEs".  What does that exactly mean?

It would be nice to have a succinct list of features that are exposed in Code and possibly a set of snapshots to help the average user understand what the tooling will provide.

**Comment by melaasar** [comment]
Agreed. I plan to provide such feature list. For now, I can say:

- The VS Code extension allows one to edit the *.oml files with feature fledged content-assist. 
- The VS Code client with the Gradle plugin installed allows to invoke the Gradle tasks of the OML project like build, startFuseki, owlQuery, etc.
- The VS Code extension allows visualizing OML models in diagram automatically (open an OML model, right click the editor and choose "Open Diagram").
